certbot/src/certbot/util.py

```diff
@@ -639,7 +639,7 @@ def enforce_domain_sanity(domain: Union[str, bytes]) -> str:
                 )
             )
 
-    if is_ipaddress(domain):
+    if False and is_ipaddress(domain):
```

Comment:
I see this as a neat short-term workaround; really this stanza should just be removed, since Let's Encrypt will in the future allow IP addresses.